examples/ffva/ffva_int_cyberon.cmake

```diff
@@ -0,0 +1,238 @@
+    #**********************
+    # Merge binaries
+    #**********************
+    merge_binaries(example_ffva_int_cyberon_${FFVA_AP} tile0_example_ffva_int_cyberon_${FFVA_AP} tile1_example_ffva_int_cyberon_${FFVA_AP} 1)
+
+    #**********************
+    # Create run and debug targets
+    #**********************
+    create_run_target(example_ffva_int_cyberon_${FFVA_AP})
+    create_debug_target(example_ffva_int_cyberon_${FFVA_AP})
+
+    #**********************
+    # Create data partition support targets
+    #**********************
+    set(TARGET_NAME example_ffva_int_cyberon_${FFVA_AP})
+    set(DATA_PARTITION_FILE ${TARGET_NAME}_data_partition.bin)
+    set(MODEL_FILE ${TARGET_NAME}_model.bin)
+    set(FATFS_FILE ${TARGET_NAME}_fat.fs)
+    set(FLASH_CAL_FILE ${LIB_QSPI_FAST_READ_ROOT_PATH}/lib_qspi_fast_read/calibration_pattern_nibble_swap.bin)
+    #set(FATFS_CONTENTS_DIR ${TARGET_NAME}_fatmktmp)
+
+
+    add_custom_target(${MODEL_FILE} ALL
+        COMMAND ${CMAKE_COMMAND} -E copy ${CYBERON_COMMAND_NET_FILE} ${MODEL_FILE}
+        COMMENT
+            "Copy Cyberon NET file"
+        VERBATIM
+    )
+
+    create_filesystem_target(
+    #[[ Target ]]                   ${TARGET_NAME}
+    #[[ Input Directory ]]          ${CMAKE_CURRENT_LIST_DIR}/filesystem_support/${MODEL_LANGUAGE}
+    #[[ Image Size ]]               ${FILESYSTEM_SIZE_BYTES}
+   )
+
+    add_custom_command(
+        OUTPUT ${DATA_PARTITION_FILE}
+        COMMAND ${CMAKE_COMMAND} -E rm -f ${DATA_PARTITION_FILE}
+        COMMAND datapartition_mkimage -v -b 1
+        -i ${FLASH_CAL_FILE}:${CALIBRATION_PATTERN_DATA_PARTITION_OFFSET} ${FATFS_FILE}:${FILESYSTEM_DATA_PARTITION_OFFSET} ${MODEL_FILE}:${MODEL_DATA_PARTITION_OFFSET}
+        -o ${DATA_PARTITION_FILE}
+        DEPENDS
+            ${MODEL_FILE}
+            make_fs_${TARGET_NAME}
+            ${FLASH_CAL_FILE}
+        COMMENT
+            "Create data partition"
+        VERBATIM
+    )
+
+    set(DATA_PARTITION_FILE_LIST
+        ${DATA_PARTITION_FILE}
+        ${MODEL_FILE}
+        ${FATFS_FILE}
+        ${FLASH_CAL_FILE}
+    )
+
+    set(DATA_PARTITION_DEPENDS_LIST
+        ${DATA_PARTITION_FILE}
+        ${MODEL_FILE}
+        make_fs_${TARGET_NAME}
+    )
+
+    # The list of files to copy and the dependency list for populating
+    # the data partition folder are identical.
+    create_data_partition_directory(
+        #[[ Target ]]                   ${TARGET_NAME}
+        #[[ Copy Files ]]               "${DATA_PARTITION_FILE_LIST}"
+        #[[ Dependencies ]]             "${DATA_PARTITION_DEPENDS_LIST}"
+    )
+
+    create_flash_app_target(
+        #[[ Target ]]                   ${TARGET_NAME}
+        #[[ Boot Partition Size ]]      ${BOOT_PARTITION_SIZE}
+        #[[ Data Partition Contents ]]  ${DATA_PARTITION_FILE}
+        #[[ Dependencies ]]             ${DATA_PARTITION_FILE}
+    )
+
+    unset(DATA_PARTITION_FILE_LIST)
+    unset(DATA_PARTITION_DEPENDS_LIST)
+
+    #add_custom_target(
```

Comment:
Do we not need the fatfs related stuff? If not then could you delete these lines?